core/station/impl/src/services/external_canister.rs

```diff
@@ -686,13 +683,9 @@ impl ExternalCanisterService {
                 match updated_call_policy.policy_id {
                     Some(policy_id) => {
                         if !current_policies.calls.contains(&policy_id) {
-                            print(format!(
-                                "Policy with id {} not found for external canister {}",
-                                Uuid::from_bytes(policy_id).hyphenated(),
-                                external_canister.canister_id.to_text()
-                            ));
-
-                            continue;
+                            return Err(ExternalCanisterError::NotFound {
```

Comment:
Do we need to return here as editing a non-existing policy later would fail anyway.

Reply:
ah right, missed that, removed then 👍